gooey/python_bindings/config_generator.py

```diff
@@ -86,6 +86,7 @@ def create_from_parser(parser, source_path, **kwargs):
       'terminal_font_family': kwargs.get('terminal_font_family', None),
       'terminal_font_weight': kwargs.get('terminal_font_weight', None),
       'terminal_font_size':   kwargs.get('terminal_font_size', None),
+      'richtext_controls':    kwargs.get('richtext_controls', False),

```

Comment:
Oh, make sure to add this to the README

Reply:
As requested, I added an entry in the README.md describing the new feature.
I also created a simple example in the examples repo and opened a PR for it :
https://github.com/chriskiehl/GooeyExamples/pull/16